tests/php/general/test_class.jetpack-network.php

```diff
@@ -89,15 +89,6 @@ public function test_get_url_returns_null_for_underspecified_subsitedisconnect()
 		$this->assertNull( $jpms->get_url(  array( 'name' => 'subsitedisconnect' ) ) );
 	}
 
-	/**
-	 * @since 2.8
-	 **/
-	public function test_set_auto_activated_modules_returns_array() {
```

Comment:
Should we port this test somewhere, or is there a good reason we're removing it?

Reply:
No need to port it, we've removed the method.